packages/wrangler/src/deploy/deploy.ts

```diff
@@ -732,6 +732,7 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 				props.assetsOptions && assetsJwt
 					? {
 							jwt: assetsJwt,
+							// TODO: routing config should be removed
 							routingConfig: props.assetsOptions.routingConfig,
```

Comment:
EWC isn't doing anything with routing config, it only cares about serve_directory, not_found_handling, and html_handling. No need to send the routing config (has_user_worker), as EWC determines this on its own. Not breaking if we do keep this, but wanted to flag for removal at some point.

Reply:
I think this gets stripped out in build-worker-multipart-formdata or some other similarly named file here

Reply:
Yup: https://github.com/cloudflare/workers-sdk/blob/f5b9cd52bc2ec42e17435c5ea0cd79b766ff76dd/packages/wrangler/src/deployment-bundle/create-worker-upload-form.ts#L207-L210

Reply:
Perfect!

Reply:
doesn't seem like we can actually remove it. That `routingConfig` is used [here](https://github.com/cloudflare/workers-sdk/blob/f5b9cd52bc2ec42e17435c5ea0cd79b766ff76dd/packages/wrangler/src/deployment-bundle/create-worker-upload-form.ts#L203) to short circuit in case static assets upload only. So we can remove this comment